src/custom_nodes/face_blur/config.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Isn't it a duplicate of what you added to demos? If so maybe remove this and link demo in the custom node readme?

Reply:
Yes it is, however each custom node has such a config.json file in their folder. So technically each CN in our repo has duplicate of config.json in demo. Is this something that we should address? Should I remove that file in such a case?

Reply:
I would have one config file in the demos and link to the demo in CN documentation, if you're okay with that.